tool/tctl/common/resource_command.go

```diff
@@ -1578,6 +1616,46 @@ func (rc *ResourceCommand) getCollection(ctx context.Context, client auth.Client
 			}
 		}
 		return &samlIDPServiceProviderCollection{serviceProviders: resources}, nil
+	case types.KindDevice:
+		remote := client.DevicesClient()
+		if rc.ref.Name != "" {
+			devices, err := findDeviceByIDOrTag(ctx, remote, rc.ref.Name, true)
```

Comment:
In the case of `allowMultiple=true`, WDYT about just calling FindDevices and returning whatever we get back, without any additional filters? I think that's likely fine here.

(If you do take this, we can drop the `allowMultiple` from the signature as well.)

Reply:
We'll get some slightly different semantics here, i.e matching both asset tag and ID at the same time if there are any collisions there. I have no strong opinion on this matter and was just an assumption. Changed.